modules/ROOT/pages/cloud-api.adoc

```diff
@@ -0,0 +1,4 @@
+= Cloud API
+:page-api-spec-url: api:ROOT:attachment$cloud-api.yaml
+:page-layout: swagger
+:page-try-it: true
```

Comment:
```suggestion
:page-try-it: true
:page-cloud-api: true
```

Reply:
Adding this attribute to accomodate https://github.com/redpanda-data/docs-ui/pull/180